documentation/converter-overview/index.html.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Flow.js Converters"
+layout: "default"
+isPage: true
+---
+
+##Flow.js: Converters
+
+
+Converters allow you to convert data -- in particular, model variables that you display in your project's user interface -- from one form to another.
```

Comment:
"For example" instead of "in particular", "values of variables in your model" instead of "model variables"